.gitmodules

```diff
@@ -6,3 +6,6 @@
 	path = applications/graph/largescale_node2vec
 	url = https://lc.llnl.gov/bitbucket/scm/havoq/largescale_node2vec.git
 	ignore = dirty
+[submodule "applications/ATOM/moses"]
+	path = applications/ATOM/moses
+	url = git@github.com:williamdjones/moses.git
```

Comment:
Should this point to Derek's github or the published one?

Reply:
Fixed, points to my fork which at this time has all the necessary scripts needed for ATOM specific dataset, these scripts were contributed by Derek.